src/tests/test_davidson.cc

```diff
@@ -164,9 +318,9 @@ Eigen::MatrixXd extract_eigenvectors(const Eigen::MatrixXd &V,
   return W;
 }
 
-class BlockOperator : public MatrixFreeOperator {
+class HermitianBlockOperator : public MatrixFreeOperator {
```

Comment:
ha ok .... should have read the entire PR before making the last comment :)

Reply:
I should read all the comments before commenting on your comments